benchmarks/util.py

```diff
@@ -41,7 +41,7 @@ def reset_rng_state(benchmark_experiment=None):
   torch.manual_seed(1337)
   random.seed(1337)
   np.random.seed(1337)
-  if benchmark_experiment and benchmark_experiment.xla:
+  if benchmark_experiment is not None and benchmark_experiment.xla:
```

Comment:
`if foo` is the same as `if foo is not None`, and AFAICT is more idiomatic. Is there a good reason not to leave this as is?

Reply:
I'm not sure what is more idiomatic but it is def not the same. `foo` will evaluate to false for a lot of things, e.g. the empty string, an empty list, ... 
With `is not None` the comparison is stricter and less ambiguous to read (?)

Reply:
Yeah I meant in this case you'll get the same result. Indeed one check is more specific than the other.

FWIW when I see "if foo is not None" or similar, that makes me think that foo might indeed be `0`, `[]`, `{}`, etc. In this case I don't think that can happen, hence my suggestion for what I believe is more idiomatic. But in any case this is a nit.